cea/scripts.yml

```
@@ -273,6 +273,31 @@ Life Cycle Analysis:
+    label: Emission - Time Dependent
+    short_description: Calculate operational and embodied emission, and their progress through time.
+    description: First, This feature generates hourly operational emission for heating, cooling, hot water
+      and electricity consumption for each building.
+      Then, this Feature accounts for all possible emissions since the building's initial construction,
+      including construction, renovation, deconstruction, biogenic carbon storage and operational emissions.
+      The timeline records all activities every year, assuming all components are renovated or replaced
+      according to its lifetime.
+    interfaces: [cli, dashboard]
+    module: cea.analysis.lca.emission_time_dependent
+    parameters: ['general:scenario', emission-time-dependent]
+    input-files: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)